packages/editor/src/browser/editor-contribution.ts

```diff
@@ -0,0 +1,51 @@
+/*
+* Copyright (C) 2017 TypeFox and others.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+import { EditorManager } from './editor-manager';
+import { injectable, inject } from "inversify";
+import { StatusbarAlignment, Statusbar } from '@theia/core/lib/browser/statusbar/statusbar';
+import { FileIconProvider } from '@theia/filesystem/lib/browser/icons/file-icons';
+import { Position } from 'vscode-languageserver-types';
+
+@injectable()
+export class EditorContribution {
```

Comment:
should be `EditorFrontendContribution` and implement `FrontendApplicationContribution`